### DISCUSSION TITLE
How can I create Cards on batch from note

### DISCUSSION THREAD

**Discussion by LumenYoung** [mgmeyers/obsidian-kanban#471] [Ideas & Feature Requests]
Hallo, I'm new to Kanban on Obsidian and find it extremely useful in my workflow. I'm using Kanban to manage my to-do in a intuitive way. 

I encounter one problem in my workflow when using Kanban. It is because I often plan a complex task in a note and decompose it into sub-tasks. The current workflow requires me to  manually create the respective sub-task cards on Kanban, so I think maybe it would be more desirable if I could directly generate tasks from the checkbox in one note. 

I think there are several ways to make this work. The first approach is to create a overall inbox for all the Kanbans. You can add card to inbox with all the todo in this note using one command, and then put cards into the kanban columns you want to place.

The second method is much more direct. We designate our object kanban column when we generate the cards. This is a much more direct approach.   

Anyway, thank you for this awesome plugin! Please keep up with this great work!

**Comment by ingvald** [comment]
there's a feature request for at least some of this: https://github.com/mgmeyers/obsidian-kanban/issues/467 . also, saw another plugin mentioned: https://github.com/mgmeyers/obsidian-kanban/discussions/525#discussioncomment-2825155
